stdlib/public/core/NormalizedCodeUnitIterator.swift

```diff
@@ -0,0 +1,253 @@
+//===--- StringNormalization.swift ----------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+internal
+struct NormalizedCodeUnitIterator: IteratorProtocol {
```

Comment:
If this becomes @_versioned, I would say it's better to underscore the name so that we reserve this name for the future. Might as well do it now.